### ISSUE TITLE
SetInterval() doesn't work properly 

### ISSUE THREAD

**Issue by jakubtelec** [kusti8/proton-native#143]
**Problem**: SetInterval() seems to work only when mouse cursor is moving inside app's window.

(also: strange comma appears in the middle of `<Text>` component)

**Os**: Windows 10 and Ubuntu 16.04

**How does it look like:**  

![SetInterval bug recording](https://image.ibb.co/fJJXT8/proton_app.gif)

**Code:**
 
```javascript
import React, { Component } from 'react'
import { render, Window, App, Text } from 'proton-native'

class Example extends Component {
  constructor(props) {
    super(props)
    this.state = {
      seconds: 0,
    }
  }

  componentDidMount() {
    this.timerID = setInterval(() => this.tick(), 1000)
  }

  componentWillUnmount() {
    clearInterval(this.timerID)
  }

  tick() {
    this.setState((prevState, props) => ({
      seconds: prevState.seconds + 1
    }))
  }
  render() {
    return (
      <App>
        <Window size={{ w: 150, h: 150 }} menuBar={false} margined>
          <Text> Timer: {this.state.seconds}</Text>
        </Window>
      </App>
    )
  }
}

render(<Example />)
```

**Comment by parro-it** [comment]
What version are you using? That problem should be solved in last version that use 0.2.0 version of libui-node

**Comment by mischnic** [comment]
>  That problem should be solved in last version that use 0.2.0 version of libui-node

No. `startLoop` is only called in `render(<Example />)`. So before, the node's `setTimeout` is used when declaring the component functions.

**Comment by parro-it** [comment]
> No. startLoop is only called in render(<Example />). So before, the node's setTimeout is used when declaring the component functions.

Oh, yes, we didn't think of this use case... I don't know how we can solve it, but the napi version should not have this problem anymore. Or we can also back port the fix in event loop to version 0.3.0

**Comment by jakubtelec** [comment]
After update SetInterval() works fine, so I'm closing the issue.
